client/src/Components/Login.jsx

```diff
@@ -1,13 +1,105 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import {
+  Link, useNavigate,
+} from 'react-router-dom';
+import { Button, TextField, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+import logo from '../assets/logo.png';
 import { useAuth } from '../useAuth';
 
+const useStyles = makeStyles({
+  head: {
+    height: '80px',
+  },
+  signInBtn: {
+
+    width: '200px',
+  },
+
+  loginContainer: {
+    display: 'flex',
+
+    height: '100vh',
+  },
+  img: {
+    height: '100%',
+    width: '50%',
+    objectFit: 'cover',
+  },
+  logo: {
+    height: '300px',
+    width: '500px',
+  },
+  formStyle: {
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    alignItems: 'center',
+    width: '100%',
+  },
+  textFld: {
+    width: '400px',
+    height: '100px',
+  },
+});
+
 function Login() {
+  const navigate = useNavigate();
   const { login } = useAuth();
-  const loginClick = () => {
-    login({ password: 'password', email: 'salma1@gmail.com' });
+  const classes = useStyles();
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+  const [errorMsg, setError] = useState(null);
+  const [data, setData] = useState({
+    email: null,
+    password: null,
+  });
+
+  const handelSubmit = (e) => {
+    e.preventDefault();
+    setData({
+      email,
+      password,
+    });
   };
+
+  const call = (error) => {
+    if (!error) {
+      navigate(`/
+      `);
+    } else {
+      setError(error.data.msg);
+    }
+  };
+
+  useEffect(() => {
+    // eslint-disable-next-line no-unused-expressions
+    data.email ? login({ password: data.password, email: data.email }, call) : null;
+  }, [data]);
```

Comment:
I think this `useEffect` is not necessary. you can add the `login` function in the `handelSubmit` function